src/main/java/gregtech/api/GTValues.java

```diff
@@ -53,6 +56,7 @@
      * The short Names for the Voltages
      */
     public static final String[] VN = new String[]{"ULV", "LV", "MV", "HV", "EV", "IV", "LuV", "ZPM", "UV", "MAX"};
+    public static final List<String> VN_LIST_LOWER_CASE = java.util.Arrays.stream(VN).map(String::toLowerCase).collect(Collectors.toList());
```

Comment:
what is this shit and why it is here

Reply:
This is part of bad implementation of getTierlessTooltipKey()